roles/nova-data/meta/main.yml

```diff
@@ -3,10 +3,11 @@ dependencies:
   - role: inspec
     install_inspec_controls:
       - nova_data
+      - kvm
     tags: inspec
     when:
       - inspec.enabled|bool
-      - inspec.controls.nova_data.enabled|bool
+      - inspec.controls.nova_data.enabled|bool or inspec.controls.kvm.enabled|bool
```

Comment:
this is faulty ...   this means it'll install both even if one of them is disabled.  may need to include the role twice,  once for each.